WebService/v1/Models/SimulationApiModel/SimulationApiModel.cs

```diff
@@ -188,6 +194,15 @@ public static SimulationApiModel FromServiceModel(Simulation value)
             }
 
             result.DeviceModels = SimulationDeviceModelRef.FromServiceModel(value.DeviceModels);
+            result.Statistics = SimulationStatistics.FromServiceModel(statistics);
+
+            if (result.Statistics != null)
+            {
+                DateTimeOffset now = DateTimeOffset.UtcNow;
+                TimeSpan duration = value.IsActiveNow ? now - value.StartTime.Value : value.StoppedTime.Value - value.StartTime.Value;
```

Comment:
is this covered by tests?

Reply:
note that it might take minutes from the StartTime to when the simulation manager starts, e.g. it can take minutes to create devices, so this metric will be quite wrong for the first 10-30 minutes of the simulation. We should consider having a timestamp in the simulation object, tracking when the simulation actually starts, which is after devices and partitions have been created

Reply:
added case for avg time.

added ActualStartTime to simulation. This will be set when actors are created for devices in a simulation. For calculating avg, we will use this time as start time.